src/test/GradleTestDevModeActions.ts

```diff
@@ -0,0 +1,268 @@
+    console.log("Attach Debugger action done");
+    await utils.delay(8000);
+    const contentPart = debugView.getContent();
+    //console.log("Get Content");
+    
+    let mysecarry: Promise<ViewSection[]> = contentPart.getSections();    
+    let mysecmap: IterableIterator<[number, ViewSection]> = (await mysecarry).entries();
+    for (const [key, value] of (mysecmap)) {
+      if ((await value.getEnclosingElement().getText()).includes("BREAKPOINTS")) {
+        //console.log("******** mysecmap getEnclosingElement " + (await value.getEnclosingElement().getText()).valueOf());
+        console.log("Found Breakpoints");
+        attachStatus = true;
+        break;
+      }
+    }
+
+    await utils.stopLibertyserver();
+    isServerRunning = !await utils.checkTerminalforServerState(constants.SERVER_STOP_STRING); //negate isServerRunning
+    if (!isServerRunning)
+      console.log("Server stopped successfully ");
+  }
+  } catch (e) {
+    console.error("error - ", e)
+  } finally {
+    console.log("finally block: is server running -  ", isServerRunning);
+    if (isServerRunning) {
+      utils.stopLibertyserver();
+    }
+    else
+      console.log("good to close test - Attach Debugger for start with custom parameter(-DdebugPort=7777) event");
+  }
+  expect(attachStatus).to.be.true;
+}).timeout(550000);
+
+it('start gradle with docker from liberty dashboard', async () => {     
+  
+  if((process.platform === 'darwin' ) || (process.platform === 'win32') || (process.platform == 'linux'))
+  {
+    //skip running for platforms , enable them for linux after resolving docker setup in GHA
+    return true;
+  }
+    
+  
+  await utils.launchDashboardAction(item, constants.START_DASHBOARD_ACTION_WITHDOCKER, constants.START_DASHBOARD_MAC_ACTION_WITHDOCKER);  
+  await utils.delay(60000);
+  const serverStartStatus = await utils.checkTerminalforServerState(constants.SERVER_START_STRING);
+  if(!serverStartStatus)
+    console.log("Server started message not found in the terminal");
+  else
+  {
+    console.log("Server succuessfully started");  
+    await utils.launchDashboardAction(item, constants.STOP_DASHBOARD_ACTION, constants.STOP_DASHBOARD_MAC_ACTION);    
+    const serverStopStatus= await utils.checkTerminalforServerState(constants.SERVER_STOP_STRING);
+    if(!serverStopStatus){ 
+    console.error("Server stopped message not found in the terminal");
+    }
+    else
+      console.log("Server stopped successfully");
+    expect (serverStopStatus).to.be.true;
+}
+ expect (serverStartStatus).to.be.true; 
+ 
+    
+}).timeout(350000);
+
+it('View test report for gradle project', async () => {      
+
+  if((process.platform === 'darwin' ) || (process.platform === 'win32') || (process.platform == 'linux'))
+  {
+    //skip running for platforms , enable them for linux after resolving docker setup in GHA
```

Comment:
how come the "View test report" test is dependent on the docker setup?

Reply:
I hv noticed that the test report results are different for gradle at different times based on the number of tabs opened and is affected based on the sequence of the tests, as number of tabs opened / results of the tabs are different. I hv asked @ragudiko to work on it and enable it . we will open a github issue to track to either resolve it / atleast update observation for anyone to take over

Reply:
issue opened --> https://github.com/OpenLiberty/liberty-tools-vscode/issues/266
The issue when number of tabs open vary can be handled by using tabs.indexOf method instead of tab index value(0 or 1)
//expect (tabs[1], "Gradle test report not found").to.equal(constants.GRADLE_TEST_REPORT_TITLE);
  expect (tabs.indexOf(constants.GRADLE_TEST_REPORT_TITLE)>-1, "Unit test report not found").to.equal(true);

Reply:
So just to be clear... this issue has nothing to do with the docker setup, correct? It's just affected by the order the tests run? If so, can the comment on line 254 be updated to refer to issue #266 instead?

Reply:
```suggestion
    //skip running for platforms , enable once https://github.com/OpenLiberty/liberty-tools-vscode/issues/266 is resolved
```